### PR TITLE
fix(frontend): `TaskSelection` component is not loading the full customer project in the first page load

### PR DESCRIPTION
**Pull request by MitanOmar**
fix: #142

### CODE REVIEW

**Comment by c0rydoras** [comment]
This can probably be done by adjusting the `include` when initially fetching the project

**Comment by c0rydoras** [comment]
Which is better as then we don't do more requests

File: frontend/app/components/task-selection/component.js [hunk 1]
```diff
@@ -116,6 +136,13 @@ export default class TaskSelectionComponent extends Component {
     } else {
       this.tracking.fetchCustomers.perform();
     }
+
+    if (this.customer?.id) {
+      this.customer.project = this.customerProjectsData.value ?? [];
+    }
+    if (this.project?.id) {
+      this.project.tasks = this.projectTasksData.value ?? [];
+    }
```
Comment (derrabauke) [inline frontend/app/components/task-selection/component.js:145]: @MitanOmar Is this copy :spaghetti:  from the `tracking` service? 
This duplicates requests and does not respect what is done in the code above. I think an include as proposed by Arthur or a fix in the `onCustomerChange` method is necessary. Why is this error surfacing and under what circumstances (please don't quote the issue)?

**Comment by c0rydoras** [comment]
ill close this as #142 has been fixed by #447

**Pull request closed without merging**